console.py

```diff
@@ -48,16 +53,16 @@ def do_show(self, line):
             print("** class name missing **")
         else:
             args = line.split(" ")
-            if args[0] != "BaseModel":
+            if args[0] not in HBNBCommand.class_list:
                 print("** class doesn't exist **")
             elif len(args) < 2:
                 print("** instance id missing **")
             else:
                 name = f'{args[0]}.{args[1]}'
                 a = 0
-                for k in models.storage.all().keys():
-                    if k == name:
-                        print(models.storage.all()[k])
+                for key, obj in storage.all().keys():
```

Comment:
if you want to work with key and value you should write it like this : for key, obj in storage.all().items():
or like how i wrote it befor to print the 'value' of the key
----
we have to use models.storage method to  avoid getting circular importing i got this error before